app/models/partner.rb

```diff
@@ -211,6 +211,26 @@ def export_json
     }
   end
 
+  def families_served
+    families.count
+  end
+
+  def children_served
+    children.count
+  end
+
+  def family_zipcodes
+    families.pluck(:guardian_zip_code).uniq.count
+  end
+
+  def impact_metrics
+    {
+      families_served: families_served,
+      children_served: children_served,
+      family_zipcodes: family_zipcodes
+    }
+  end
```

Comment:
Let's make sure to add a spec for this one. Lemme know if you need help or clarification on this one :)